intermediate_source/realtime_rpi.rst

```diff
@@ -68,6 +68,7 @@ PyTorch and all the other libraries we need have ARM 64-bit/aarch64 variants so
 .. code:: shell
 
     $ pip install torch torchvision torchaudio
+    # Installing OpenCV will take about 2 hours and nothing shows in the terminal until it finishes.
```

Comment:
@d4l3k is this expected?

Reply:
That seems way longer than expected. @YunjinJo  what RPi are you using? Are you using 64bit arm or 32bit? And it's the install that takes a long time not the download?

Reply:
Though, if it builds from the source distribution that might be reasonable... 

Seems like other folks are seeing the same thing: https://raspberrypi-guide.github.io/programming/install-opencv

There's some prebuild rpi wheels at https://www.piwheels.org/project/opencv-contrib-python/ but only for 32bit